.gitignore

```diff
@@ -99,3 +99,10 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# config/tweet files
+config.json
+turn.txt
+tweets.txt
+next_tweet_index.txt
+tweetme.log
```

Comment:
Ideally it should be generated somewhere under `/etc/` but I think it is okay for now.

Reply:
We can shift them if there are enough files for us to handle. Right now the repository looks clean since all these files are ignored by default.